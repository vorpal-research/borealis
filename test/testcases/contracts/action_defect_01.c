#include "defines.h"

// @requires argc > 13
int main(int argc, char** argv) {

	borealis_action_defect("BUG-42", "Lorem ipsum");

	if (argc <= 13) {
		borealis_action_defect("WTF-13", "Should not happen");
	}

	return 0;
}
