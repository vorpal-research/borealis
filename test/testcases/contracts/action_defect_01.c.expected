
[
	
	{
		"location" : 
		{
			"filename" : "test/testcases/contracts/action_defect_01.c",
			"loc" : 
			{
				"col" : 2,
				"line" : 6
			}
		},
		"type" : "BUG-42"
	}
]
